<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>176</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DTG</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>20</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Pulser</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_pulsesep">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>160</y>
      <width>68</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>5.000000000000000</double>
    </property>
    <property name="maximum">
     <double>200.000000000000000</double>
    </property>
    <property name="value">
     <double>50.000000000000000</double>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_run">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>260</y>
      <width>105</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox_load"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_load">
         <property name="text">
          <string>load seq</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_from_file">
         <property name="text">
          <string>from file</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_pulselen">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>110</y>
      <width>68</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>150.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_pulsesep">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>140</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pulse separation, ns</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_load">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>240</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>load sequence</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pulselen">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>90</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pulse length, ns</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_write">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_continuous">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CW</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_channel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_pulsesep_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>channel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
